### PR TITLE
Profile modal

### PR DESCRIPTION
**Pull request by ivinayakg**
https://vide-x.netlify.app/

### CODE REVIEW

**Review by jeetbhalani15 — APPROVED** [review]
- modal works fine
-  ui looks good

File: src/components/ProfileModal/index.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+          <button className={classes.btn} onClick={logoutHandler}>
+            <i className="fas fa-sign-out-alt"></i>
+            <h3>Logout</h3>
+          </button>
+          <button className={classes.btn} onClick={LikedClickHanlder}>
+            <i className="fas fa-thumbs-up"></i>
+            <h3>Liked Videos</h3>
+          </button>
+          <button className={classes.btn} onClick={PlaylistClickHanlder}>
+            <i className="fas fa-photo-video"></i>
```
Comment (ark2002) [inline src/components/ProfileModal/index.js:71]: typos here for handlers

**Review by ravikumar0403 — APPROVED** [review]
Code is clean and readable
Liked the usage of custom hooks, css modules and React portals

File: src/components/ProfileModal/index.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  };
+  const LikedClickHanlder = () => {
+    navigate("/user/likedVideos");
+    setShow(false);
+  };
+  const PlaylistClickHanlder = () => {
+    navigate("/user/playlists");
+    setShow(false);
+  };
+
```
Comment (ark2002) [inline src/components/ProfileModal/index.js:41]: from  here

**Review by ark2002 — APPROVED** [review]
rest looks great

**Pull request merged**